docs/data/material/components/floating-action-button/FloatingActionButtonExtendedSize.js

```diff
@@ -6,15 +6,15 @@ import NavigationIcon from '@mui/icons-material/Navigation';
 export default function FloatingActionButtonExtendedSize() {
   return (
     <Box sx={{ '& > :not(style)': { m: 1 } }}>
-      <Fab variant="extended" size="small" color="primary" aria-label="add">
+      <Fab variant="extended" size="small" color="primary" aria-label="extended">
         <NavigationIcon sx={{ mr: 1 }} />
         Extended
       </Fab>
-      <Fab variant="extended" size="medium" color="primary" aria-label="add">
+      <Fab variant="extended" size="medium" color="primary" aria-label="extended">
         <NavigationIcon sx={{ mr: 1 }} />
         Extended
       </Fab>
-      <Fab variant="extended" color="primary" aria-label="add">
+      <Fab variant="extended" color="primary" aria-label="extended">
```

Comment:
Labels are for what the widget does, not what the demo is for. This button was for adding something so the original label made sense.

Reply:
Code reverted to original one